resources/l10n/zh_CN/LC_MESSAGES/loot.po

```diff
@@ -1,2167 +1,921 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: Better Oblivion Mod Manager 2.1\n"
+"Project-Id-Version: LOOT v0.7\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2013-10-26 11:41-0000\n"
-"PO-Revision-Date: 2014-02-04 20:21-0000\n"
-"Last-Translator: icedream <icedream@tesfans.org>\n"
-"Language-Team: 上古卷轴爱好者 <icedream@tesfans.org>\n"
-"Language: zh\n"
+"POT-Creation-Date: 2015-01-09 10:40-0000\n"
+"PO-Revision-Date: 2015-02-07 13:08+0800\n"
+"Last-Translator: S猫 <silentdark@skycitizen.net>\n"
+"Language-Team: 天际公民 <silentdark@skycitizen.net>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 1.6.3\n"
-
-#: backend/game.cpp:96
+"X-Generator: Poedit 1.7.3\n"
+"X-Poedit-KeywordsList: translate:1,1t;translate:1c,2,t\n"
+"X-Poedit-Basepath: ../../\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: zh_CN\n"
+"X-Poedit-SearchPath-0: src/backend\n"
+"X-Poedit-SearchPath-1: src/gui\n"
+"X-Poedit-SearchPath-2: resources/report/js\n"
+
+#: src/backend/game.cpp:347
 msgid "Invalid game ID supplied."
 msgstr "无效的游戏ID"
 
-#: backend/game.cpp:147
+#: src/backend/game.cpp:371
 msgid "Game path could not be detected."
 msgstr "无法检测到游戏路径。"
 
-#: backend/game.cpp:248 backend/game.cpp:308 backend/game.cpp:363
+#: src/backend/game.cpp:489
+msgid ""
+"libloadorder failed to create a game handle. Details could not be fetched."
+msgstr "加载顺序数据库无法创建游戏处理器，详细信息无法获取。"
+
+#: src/backend/game.cpp:493
 msgid "libloadorder failed to create a game handle. Details:"
-msgstr "libloadorder 无法创建游戏处理器，详细信息："
+msgstr "加载顺序数据库无法创建游戏处理器，详细信息："
```

Comment:
`libloadorder` is a "proper noun", it is the name of a specific software library, so it shouldn't be translated.